integration_tests/suite/base/test_event_handlers.py

```diff
@@ -35,4 +36,9 @@ def templates_created():
             ),
         )
 
+    def slut_created():
+        # There's no API to check the tenant slug. It is part of any new group name
+        group = confd.groups.create({'label': 'ignore'}, wazo_tenant=CREATED_TENANT)
+        assert_that(group, has_entries(name=starts_with('grp-myslug-')))
+
     until.assert_(templates_created, tries=5)
```

Comment:
```suggestion
    def slut_created():
        # There's no API to check the tenant slug. It is part of any new group name
        group = confd.groups.create({'label': 'ignore'}, wazo_tenant=CREATED_TENANT)
        assert_that(group, has_entries(name=starts_with('grp-myslug-')))

    until.assert_(templates_created, tries=5)
    until.assert_(slug_created, tries=5)
```

Reply:
Done, I also fixed the `slug_created` typo